vendor/github.com/docker/libnetwork/drivers/windows/overlay/ov_endpoint_windows.go

```diff
@@ -263,3 +269,14 @@ func (d *driver) EndpointOperInfo(nid, eid string) (map[string]interface{}, erro
 
 	return data, nil
 }
+
+func endpointRequest(method, path, request string) (*hcsshim.HNSEndpoint, error) {
+	if windowsBuild == 14393 {
```

Comment:
FWIW, I think this fix should be upstreamed to hcsshim; this is not something consumers of the hcsshim package should have to deal with; I'll have a look if I can create a patch there

Reply:
Yes. We discussed about it. Unfortunately, this pulls in a lot of vendor code which requires more  integration test cases. Given the immediate release concerns, this is a good compromise to have.